packages/devtools_app/lib/src/screens/vm_developer/vm_class_screen_controller.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/foundation.dart';
+import 'package:vm_service/vm_service.dart';
+
+import '../../primitives/auto_dispose.dart';
+import '../../service/vm_service_wrapper.dart';
+import '../../shared/globals.dart';
+import '../debugger/debugger_model.dart';
+
+/// Auto DisposableController for fetching and displaying data on the class screen.
+class ClassScreenController extends DisposableController
+    with AutoDisposeControllerMixin {
+  ClassScreenController(ObjRef? objRef) {
```

Comment:
I'd change the type here to `ClassRef` so that the cast has to happen outside of this object as this object will never be able to handle anything other than `ClassRef` inputs.